package.json

```diff
@@ -1,54 +1,81 @@
 {
-  "name": "melika-ui",
-  "version": "0.1.0",
+  "name": "melika",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
   "private": true,
-  "engines": {
-    "node": "^8.9.0 || >=9.10.0"
+  "scripts": {
+    "dev": "webpack-dev-server --port 4789 --hot --inline --watch --mode development",
+    "build": "webpack --mode production"
```

Comment:
We can do it later, but we will need a `test` and a `lint` script

Reply:
So I implemented prettier - once you yarn install you should start to see files fixed once you save based on your prettier/eslint settings.